packages/tc-ui/src/primitives/relationship/__tests__/e2e-create-relationships.cyp.js

```diff
@@ -23,6 +23,121 @@ describe('End-to-end - relationship creation', () => {
 		save();
 	});
 
+	describe('one-to-one relationship', () => {
+		it('can select a selection for one-to-one relationship', () => {
+			visitMainTypePage();
+			visitEditPage();
+
+			pickFavouriteChild();
+			save();
+
+			cy.url().should('contain', `/MainType/${code}`);
+			cy.get('#favouriteChild')
+				.should('have.text', `${code}-first-child`)
+				.should('have.attr', 'href', `/ChildType/${code}-first-child`);
+		});
+
+		it('can not select another selection for one-to-one relationship', () => {
```

Comment:
I think there are 3 things to test here
- is it disabled on page load if relationship already exists
- does it get undisabled if relationship is removed
- does it get re-enabled if relationship is added again

Not sure how to structure the tests, but these are all important cases